src/ooxml/java/org/apache/poi/ooxml/util/PackageHelper.java

```diff
@@ -121,17 +118,17 @@ private static void copy(OPCPackage pkg, PackagePart part, OPCPackage tgt, Packa
      * @param tgt target properties
      */
     private static void copyProperties(PackageProperties src, PackageProperties tgt){
-        tgt.setCategoryProperty(src.getCategoryProperty().getValue());
-        tgt.setContentStatusProperty(src.getContentStatusProperty().getValue());
-        tgt.setContentTypeProperty(src.getContentTypeProperty().getValue());
-        tgt.setCreatorProperty(src.getCreatorProperty().getValue());
-        tgt.setDescriptionProperty(src.getDescriptionProperty().getValue());
-        tgt.setIdentifierProperty(src.getIdentifierProperty().getValue());
-        tgt.setKeywordsProperty(src.getKeywordsProperty().getValue());
-        tgt.setLanguageProperty(src.getLanguageProperty().getValue());
-        tgt.setRevisionProperty(src.getRevisionProperty().getValue());
-        tgt.setSubjectProperty(src.getSubjectProperty().getValue());
-        tgt.setTitleProperty(src.getTitleProperty().getValue());
-        tgt.setVersionProperty(src.getVersionProperty().getValue());
+        tgt.setCategoryProperty(src.getCategoryProperty().orElse(null));
+        tgt.setContentStatusProperty(src.getContentStatusProperty().orElse(null));
+        tgt.setContentTypeProperty(src.getContentTypeProperty().orElse(null));
+        tgt.setCreatorProperty(src.getCreatorProperty().orElse(null));
+        tgt.setDescriptionProperty(src.getDescriptionProperty().orElse(null));
+        tgt.setIdentifierProperty(src.getIdentifierProperty().orElse(null));
+        tgt.setKeywordsProperty(src.getKeywordsProperty().orElse(null));
+        tgt.setLanguageProperty(src.getLanguageProperty().orElse(null));
+        tgt.setRevisionProperty(src.getRevisionProperty().orElse(null));
+        tgt.setSubjectProperty(src.getSubjectProperty().orElse(null));
+        tgt.setTitleProperty(src.getTitleProperty().orElse(null));
+        tgt.setVersionProperty(src.getVersionProperty().orElse(null));
```

Comment:
It seems that the setters expect an `Optional`, not a potentially `null` value.

Reply:
I agree that `tgt.set*Property(String category)` could take Optional<String> but that is a bigger change. Maybe I could support the existing set methods but also add the overloaded methods that support Optional.

Reply:
I added the new API methods so was able to apply the requested change